lib/components/Modal.js

```diff
@@ -101,6 +103,8 @@ export default class Modal extends Component {
   }
 
   componentWillUnmount () {
+    if (!this.mounted) return;
```

Comment:
You can check if `this.node` is already filled instead of creating a flag to notify when it's mounted.